docs/get_started/plan_deployment/traditional_deployment/cluster_consideration/clus_maint.md

```diff
@@ -1,30 +1,32 @@
+Apply all minor fixes on each cluster node. Use the installation instructions that come with the fix. When you apply minor fixes that might update previously deployed enterprise applications<!-- How would users determine that a fix might update in this manner? -->, first turn off the auto-synchronization feature of the deployment manager. After you deploy the fix on all cluster nodes, you can force a manual synchronization with the deployment manager to ensure that all updates are synchronized on the nodes. Then, you can enable the auto-synchronization feature again.
 
-If the documentation associated with the minor fix requires that you restart HCL Portal or WebSphere Application Server, apply the minor fix one node at a time. This process enables other nodes to continue to provide service to your users. However, if the fix requires an update to the HCL Portal databases, you might be required to stop the cluster before you apply the fix. If so, use a procedure that ensures 24x7 availability.
+If the documentation associated with the minor fix requires that you restart HCL Portal or WebSphere Application Server, apply the minor fix one node at a time. This process enables other nodes to continue to provide service to users. However, if the fix requires an update to the HCL Portal databases, you might be required to stop the cluster before you apply the fix. If so, use a procedure that ensures 24x7 availability.
 
 ## Fixes and service packs
 
-There are multiple approaches to installing service packs into an HCL Portal clustered environment. Installing a service pack involves modifying the IP sprayer to remove a cluster from receiving user requests. This process allows time to finish handling existing user sessions and to upgrading that cluster on all nodes. After verification testing assures that the upgraded cluster is operational, it can start receiving production traffic while the next cluster is taken offline and goes through the upgrade process. This approach preserves complete 24x7 availability during the upgrade process.
+You can approach installing service packs into an HCL Portal clustered environment several ways. Installing a service pack involves modifying the IP sprayer to remove a cluster from receiving user requests. By using this process, you offer the system time to finish handling existing user sessions and to upgrade that cluster on all nodes. After verification testing assures that the upgraded cluster is operates, it can start receiving production traffic while the next cluster is taken offline and goes through the upgrade process. This approach preserves complete 24x7 availability during the upgrade process.
```

Comment:
```suggestion
You can approach installing service packs into an HCL Portal clustered environment in several ways. Installing a service pack involves modifying the IP sprayer to remove a cluster from receiving user requests. By using this process, you offer the system time to finish handling existing user sessions and to upgrade that cluster on all nodes. After verification testing assures that the upgraded cluster is operational, it can start receiving production traffic while the next cluster is taken offline and goes through the upgrade process. This approach preserves complete 24x7 availability during the upgrade process.
```